pep-0670.rst

```diff
@@ -284,6 +284,41 @@ single long line. Inside the function, the *op* argument has a well
 defined type: ``PyObject*``.
 
 
+Macros converted to functions since Python 3.8
+==============================================
+
+Macros converted to static inline functions
+-------------------------------------------
+
+* ``Py_INCREF()``, ``Py_DECREF()``, ``Py_XINCREF()``, ``Py_XDECREF()``:
+  Python 3.8
+* ``PyObject_INIT()``, ``PyObject_INIT_VAR()``: Python 3.8
+* Private functions: ``_PyObject_GC_TRACK()``,
+  ``_PyObject_GC_UNTRACK()``, ``_Py_Dealloc()``: Python 3.8
+* ``Py_REFCNT()``: Python 3.10
+* ``Py_TYPE()``, ``Py_SIZE()``: Python 3.11
```

Comment:
I think the formatting should match the formatting of the _"Macros converted to regular functions"_ sub-section. Something like this:

```suggestion

Python 3.8:

* ``Py_DECREF()``
* ``Py_INCREF()``
* ``Py_XDECREF()``
* ``Py_XINCREF()``
* ``PyObject_INIT()``
* ``PyObject_INIT_VAR()``
* ``_PyObject_GC_UNTRACK()``
* ``_Py_Dealloc()``

Python 3.10:

* ``Py_REFCNT()``

Python 3.11:

* ``Py_TYPE()``
* ``Py_SIZE()``
```